shared/src/iosMain/kotlin/co/touchlab/kampkit/NativeViewModel.kt

```diff
@@ -24,27 +24,27 @@ class NativeViewModel(
     init {
         ensureNeverFrozen()
         breedModel = BreedModel()
-        observeBreeds()
+        getBreeds()
     }
 
-    private fun observeBreeds() {
+    fun getBreeds() {
         scope.launch {
             log.v { "Observe Breeds" }
-            breedModel.selectAllBreeds()
+            breedModel.getBreeds()
                 .collect { summary ->
                     log.v { "Collecting Things" }
                     viewUpdate(summary)
                 }
         }
     }
 
-    fun getBreedsFromNetwork() {
+/*    fun getBreedsFromNetwork() {
```

Comment:
If I remember correctly this was put in place to force getting new breeds from the network when pulling down, but I could be completely wrong, it's been a while. If it's not in there then ignore this comment.